src/pkg/packager/interactive.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/zarf-dev/zarf/src/pkg/utils"
 )
 
+// TODO(mkcp): Implementation is heavily dependent on Message
```

Comment:
I don't see anything in here we can't remove, everything only shows up when the `--legacy` flag is used

Reply:
🫡 sendin it